exercise_01.py

```diff
@@ -0,0 +1,19 @@
+def display_pattern(n):
+    n_to_print = ""
+    total = ""
+    for i in range(1, n + 1):
+        n_to_print = "{} {}".format(i, n_to_print)
+        space = "  " * (n - i)
+
+        total =  "{}\n{}{}".format(total, space, n_to_print)
```

Comment:
could you use an f-string, don't let [Mariatta](https://twitter.com/mariatta) down! She promotes people to use f-strings.